xbmc/music/MusicDatabase.cpp

```diff
@@ -2057,7 +2070,7 @@ bool CMusicDatabase::GetArtist(int idArtist, CArtist& artist, bool fetchAll /* =
       m_pDS->close();
       return false;
     }
-
+    strSQL2 = strSQL + " - ";
```

Comment:
Use of strSQL2 sent me expecting use of 2 datasets, then this had me scratching my head for the SQL.
Since it is just text for the debug log statement can you rename it and add an inline comment. Now we no longer have _FUNCTION_ that at least made logs from Windows easier to debug, I have to wonder is soem other text would be better than outputing the  SQL string.